src/cli/command-package.ts

```diff
@@ -85,10 +84,8 @@ export default async function run(options: CliOptions) {
         }]))['npm-version'] as string;
   }
 
-  console.log(
-      chalk.dim('[1/2]') + ' 🌀  ' + chalk.magenta(`Converting Package...`));
+  logStep('[1/2]', '🌀', `Converting Package...`);
```

Comment:
Eventually it might be nice to use (or build a simple one) a task runner API that does this.